qa/rpc-tests/mempool_size_limit.py

```diff
@@ -194,28 +270,33 @@ def create_chained_transactions(node, tmpdir, utxos):
     signresult = node.signrawtransaction(rawtx, None, None, "NONE")
     csize = len(signresult['hex']) // 2
     feerate = low_fee / csize
+    assert(abs(csize - EXP_SIZE) <= 1)
 
-    write_to_file(tmpdir + "/ctxs/ctx0", feerate, signresult["hex"])
-    txs.append({'feerate': high_fee / csize, 'hex': signresult["hex"]})
+    write_to_file(tmpdir + "/tx0", feerate, signresult["hex"])
+    txs.append({'feerate': feerate, 'hex': signresult["hex"]})
```

Comment:
Just a check, not a real thing to point out. The fact the we used `high_fee` before the update was less correct than what happens now?

Reply:
Indeed, the previous version was not consistent, as `tx0` was actually using `low_fee`, and not `high_fee`. It is correct to use `low_fee` for the first transaction (which by itself becomes a good eviction candidate), and `high_fee` for the second one (which increases the feerate of the dependencies chain of `tx0`).

Reply:
Clear! Thanks for explaining!